packages/engine-server/src/doctor/service.ts

```diff
@@ -380,7 +380,8 @@ export class DoctorService implements Disposable {
           note.id = genUUID();
           await engine.writeNote(note, {
             runHooks: false,
-            updateExisting: true,
+            // Old note needs to be removed
```

Comment:
Is this change related to this PR?

Reply:
Needed to make this change to make one of the tests to pass which expected note ids to change.
In addition, before this change, the old note would still exist in the engine.